lib/routes/home/wallet_dashboard.dart

```diff
@@ -114,7 +102,38 @@ class WalletDashboardState extends State<WalletDashboard> {
                                               widget._offsetFactor)))
                   : SizedBox(),
             ),
-          )
+          ),
+          Positioned(
+              top: 70 - BALANCE_OFFSET_TRANSITION * widget._offsetFactor,
+              child: Center(
+                child: widget._accountModel != null &&
+                        !widget._accountModel.initial &&
+                        (widget._accountModel.fiatConversionList.isNotEmpty ||
+                            widget._accountModel.fiatCurrency != null) &&
+                        isAboveMinAmount(widget._accountModel.fiatCurrency)
+                    ? FlatButton(
+                        onPressed: () {
+                          var nextFiatCurrencyIndex = widget
+                                  ._accountModel.fiatConversionList
+                                  .indexOf(widget._accountModel.fiatCurrency) +
+                              1;
```

Comment:
findNextValidFiatConversion is a method of this class and has access to the widget._accountModel. I think it is better to move all the logic there and keep this UI section simpler.

Reply:
This logic is moved to findNextValidFiatConversion(nextValidFiatConversion) method https://github.com/breez/breezmobile/pull/338/commits/eb9e3c34ea346715bf0efa1572f535f5428933ec